targets/XPlatCppSdk/source/cppsdk/include/playfab/QoS/PlayFabQoSApi.h

```diff
@@ -0,0 +1,53 @@
+#pragma once
+
+#include <future>
+#include <chrono>
+
+#include <playfab/QoS/QoS.h>
+#include <playfab/QoS/QoSResult.h>
+#include <playfab/QoS/QoSSocket.h>
+#include <playfab/PlayFabMultiplayerDataModels.h>
+#include <playfab/PlayFabEventsDataModels.h>
+#include <playfab/PlayFabError.h>
+
+namespace PlayFab
+{
+    namespace QoS
+    {
+        PingResult GetQoSResultForDatacenter(std::shared_ptr<QoSSocket> socket);
```

Comment:
>GetQoSResultForDatacenter [](start = 19, length = 25)

Since this function is global, please add a comment above it explaining what this function is for and whether it is meant to be called by user